### PR TITLE
docs: revised the SDK integration steps in the getting started section

### PR DESCRIPTION
**Pull request by zhu-xiaowei**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

1. revised the SDK integration steps in the getting started section

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [ ] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting
  - [ ] streaming ingestion
    - [ ] with Redshift Serverless
    - [ ] with provisioned Redshift

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

**Review by luorobin-a2z — APPROVED** [review]

**Comment by zxkane** [comment]
### SonarQube Quality Gate Result
- **Result**: :white_check_mark: OK
- Triggered by @zhu-xiaowei on `pull_request`

| Metric | Status | Value | Error Threshold |
|:------:|:------:|:-----:|:---------------:|
|Reliability rating|:white_check_mark: OK|1|> 1|
|Security rating|:white_check_mark: OK|1|> 1|
|Sqale rating|:white_check_mark: OK|1|> 1|
|Coverage|:white_check_mark: OK|83.50|< 80|
|Duplicated lines density|:white_check_mark: OK|8.50|> 30|
|Blocker violations|:white_check_mark: OK|0|> 0|
|Bugs|:white_check_mark: OK|0|> 0|
|Code smells|:white_check_mark: OK|9|> 40|
|Critical violations|:white_check_mark: OK|0|> 0|
|Major violations|:white_check_mark: OK|0|> 0|
|Vulnerabilities|:white_check_mark: OK|0|> 0|

[View on SonarQube](http://sonarqube:9000/dashboard?id=pr-1514)
###### _updated: 7/5/2024, 05:34:52 (UTC+0)_

**Comment by zxkane** [comment]
# SonarQube Code Analytics
## Quality Gate passed

passed

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
bug   [0 Bugs](http://sonarqube:9000/project/issues?id=pr-1514&resolved=false&sinceLeakPeriod=true&types=BUG)

vulnerability   [0 Vulnerabilities](http://sonarqube:9000/project/issues?id=pr-1514&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY)

code_smell   [0 Code Smells](http://sonarqube:9000/project/issues?id=pr-1514&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL)

## Coverage and Duplications
*No data* **Coverage**

*No data* **Duplication**

File: docs/zh/getting-started/3.integrate-sdk.md [hunk 1]
```diff
@@ -55,10 +55,36 @@
...
+在刷新历史数据之前，我们需要确保之前发送的事件都已加载到 Redshift 中。
+
+1. 打开您的 Project 详细信息页面，单击 **View Details**（查看详细信息）按钮，然后进入您的项目的管道详细信息页面。
+2. 在管道详细信息页面中，单击并打开 **Monitoring**（监控）选项卡，然后单击 **View in CloudWatch**（在 CloudWatch 中查看） 按钮以进入解决方案 CloudWatch 仪表板。
+3. 向下滚动并找到 CloudWatch 仪表板中的 **Data Modeling - Redshift Serverless** 部分，然后确保 **'Load data to redshift tables' workflow** 表中的所有状态变为 **ExecutionsSucceeded**。
+
+### 刷新历史数据
+将所有事件加载到 Redshift 后，我们可以按照以下步骤刷新历史数据。
+
+1. 在数据管道所在区域的 AWS 控制台中打开 [Step Functions](https://ap-northeast-1.console.aws.amazon.com/states/home#/statemachines) 服务。
```
Comment (zxkane) [inline docs/zh/getting-started/3.integrate-sdk.md:75]: don't pin a region or remove the link

**Review by zxkane — CHANGES_REQUESTED** [review]

**Comment by zxkane** [comment]
Also note, the changes on branch `main` won't go live.

File: docs/zh/getting-started/3.integrate-sdk.md (hunk 1 above)
Comment (zhu-xiaowei) [reply docs/zh/getting-started/3.integrate-sdk.md]: done.

**Comment by zhu-xiaowei** [comment]
> , the changes on branch `main` won't go live.

Okay, I will also cherry pick these changes to v1.1.x

**Comment by zxkane** [comment]
### SonarQube Quality Gate Result
- **Result**: :white_check_mark: OK
- Triggered by @zhu-xiaowei on `pull_request`

| Metric | Status | Value | Error Threshold |
|:------:|:------:|:-----:|:---------------:|
|Reliability rating|:white_check_mark: OK|1|> 1|
|Security rating|:white_check_mark: OK|1|> 1|
|Sqale rating|:white_check_mark: OK|1|> 1|
|Coverage|:white_check_mark: OK|83.50|< 80|
|Duplicated lines density|:white_check_mark: OK|8.50|> 30|
|Blocker violations|:white_check_mark: OK|0|> 0|
|Bugs|:white_check_mark: OK|0|> 0|
|Code smells|:white_check_mark: OK|9|> 40|
|Critical violations|:white_check_mark: OK|0|> 0|
|Major violations|:white_check_mark: OK|0|> 0|
|Vulnerabilities|:white_check_mark: OK|0|> 0|

[View on SonarQube](http://sonarqube:9000/dashboard?id=pr-1514)
###### _updated: 7/5/2024, 12:19:27 (UTC+0)_

**Review by zxkane — APPROVED** [review]

**Pull request merged**